README.md

```diff
@@ -0,0 +1,227 @@
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="AXCELL EVANS" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>My new blog</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+<!-- this is my new blog I created with HTML, CSS -->
+
+> Describe your project in 1 or 2 sentences.
+
+**[Hello Microverse]** is a...
+1- HTML
+2- css
+3- linters
+
```

Comment:
This is the second time I am reviewing your code and you have not worked on this issue. Please always submit the pull request where changes have been suggested, this helps the next reviewer see the history of what you are working on and better help you get approval. 

Kindly fix this issue.
- [ ] You have not added a proper title and description for your project in the README file. Kindly edit this. See how it should be below 👇🏾 

Your work | How it should be
---- | ----
![image](https://user-images.githubusercontent.com/84156714/226848638-94642074-61c7-4444-b03d-600de9e12a4e.png) | ![image](https://user-images.githubusercontent.com/84156714/226848547-5e397773-73a9-44a2-8aae-c8fe9807d419.png)
-- | The title should be without the square brackets **[ ]**


The title - **Hello Microverse** should be preceded by the `#` to make it an `h1` header.